tools/watch_logs.py

```diff
@@ -10,6 +10,44 @@
 import time
 import threading
 from baselayer.log import colorize
+import logging
+import socket
+from logging.handlers import SysLogHandler
+
+from app.env import load_env
+
+env, cfg = load_env()
+
+
+# check 3rd party logging, if enabled, and make sure that
+# it is set up properly
+if cfg.get('external_logging'):
+    external_logging_enabled = cfg['external_logging']["enabled"]
+    if external_logging_enabled:
+        if cfg['external_logging']['service_name'] != 'papertrail':
+            # we do not know how to do this yet
+            print(
+                "Warning: we only know how to use papertrail"
+                " for external logging"
+            )
+            external_logging_enabled = False
+        try:
+            if cfg['external_logging']["url"].find("papertrailapp.com") == -1:
+                print("Warning: incorrect URL for papertrail logging")
```

Comment:
Why not use baselayer.log (as in other PRs) here and below?

Reply:
Done.